changelog.d/20250324_095007_PL-133563-redis-remove-ifd_scriv.md

```diff
@@ -0,0 +1,25 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+- If you access `services.redis."".password` in your NixOS config, please change this to
+  `flyingcircus.services.redis.password`.
```

Comment:
Just for the record, as we're releasing this during an ongoing 24.11 release cycle: If you don't change this (or do this later), nothing breaks but you continue to get longer eval times due to IFD?